storage/rocksdb/ha_rocksdb.cc

```diff
@@ -1387,6 +1392,10 @@ static MYSQL_SYSVAR_ULONG(
     nullptr, nullptr, rocksdb_persistent_cache_size_mb,
     /* min */ 0L, /* max */ ULONG_MAX, 0);
 
+static MYSQL_SYSVAR_STR(wsenv_path, rocksdb_wsenv_path,
```

Comment:
I'm not sure if we are building with this enabled or not, but if o we will need documentation ticket for it.

Reply:
We will not use it. I kept it just to minimize a diff with upstream.